AnkiDroid/src/main/java/com/ichi2/ui/AutoSizeCheckBoxPreference.kt

```diff
@@ -13,41 +13,44 @@
  You should have received a copy of the GNU General Public License along with
  this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+@file:Suppress("DEPRECATION")
+
 package com.ichi2.ui
 
 import android.content.Context
+import android.preference.CheckBoxPreference
 import android.util.AttributeSet
 import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
-import com.ichi2.utils.KotlinCleanup
 import com.ichi2.utils.ViewGroupUtils.getAllChildren
 
 // extending androidx.preference didn't work:
 // java.lang.ClassCastException: com.ichi2.ui.AutoSizeCheckBoxPreference cannot be cast to android.preference.Preference
-@Suppress("deprecation") // TODO Tracked in https://github.com/ankidroid/Anki-Android/issues/5019
-@KotlinCleanup("auto-lint (except unused class)")
-class AutoSizeCheckBoxPreference : android.preference.CheckBoxPreference {
-    @Suppress("unused") constructor(context: Context?) : super(context) {}
-    @Suppress("unused") constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {}
-    @Suppress("unused") constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {}
-    @Suppress("unused") constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int, defStyleRes: Int) : super(context, attrs, defStyleAttr, defStyleRes) {}
+// TODO Tracked in https://github.com/ankidroid/Anki-Android/issues/5019
+
+@Suppress("DEPRECATION")
```

Comment:
Why was this changed?

```suggestion
@Suppress("deprecation") // TODO Tracked in https://github.com/ankidroid/Anki-Android/issues/5019
```